test/YesSql.Tests/project.json

```diff
@@ -19,7 +19,7 @@
       "dependencies": {
         "NETStandard.Library": "1.5.0-*",
         "dotnet-test-xunit": "1.0.0-dev-*",
```

Comment:
@sebastienros @Skrypt It isn't correct 
`"dotnet-test-xunit": "1.0.0-rc2-*"` or `"dotnet-test-xunit": "1.0.0-rc2-173384-19"` 

More information: [aspnetcidev feed](http://myget.org/feed/aspnetcidev/package/nuget/dotnet-test-xunit) or [aspnetcirelease feed](http://myget.org/feed/aspnetcirelease/package/nuget/dotnet-test-xunit)